tests/unit/test_config.py

```diff
@@ -750,3 +750,43 @@ def test_get_local_specfile_path():
 def test_notifications_section():
     pc = PackageConfig.get_from_dict({"specfile_path": "package.spec"})
     assert pc.notifications.pull_request.successful_build
+
+
+@pytest.mark.parametrize(
+    "package_config, projects_list",
+    [
+        (
+            PackageConfig(
+                specfile_path="xxx",
+                jobs=[
+                    JobConfig(
+                        type=JobType.copr_build,
+                        trigger=JobConfigTriggerType.pull_request,
+                        metadata={},
+                    )
+                ],
+            ),
+            [],
+        ),
+        (
+            PackageConfig(
+                specfile_path="xxx",
+                jobs=[
+                    JobConfig(
+                        type=JobType.copr_build,
+                        trigger=JobConfigTriggerType.pull_request,
+                        metadata={"project": "example"},
+                    )
+                ],
+            ),
+            ["example"],
+        ),
+    ],
+)
+def test_project_from_copr_build_job(package_config, projects_list):
+    config_project_value = [
+        job.metadata.get("project")
+        for job in package_config.jobs
+        if job.type == JobType.copr_build and job.metadata.get("project")
+    ]
```

Comment:
It would be nice to share the code with the real usage so we don't need to edit multiple places in case of a change.

Reply:
could we have it as a method on package_config class? that would be nice